lib/training-module.js

```diff
@@ -0,0 +1,48 @@
+const EntityMigrator = require('./entity-migrator');
+
+class TrainingModule extends EntityMigrator {
+
+  get id() {
+    return this.inbound.training_module_id;
+  }
+
+  constructor(data, db, options) {
+    super(data, db, options);
+    this.model = db.TrainingModule;
+  }
+
+  static transform(data) {
+    return {
+      establishmentId: data.training_module_establishment,
```

Comment:
This is redundant as there's no association between a training module and an establishment.